8.2.16 Release # DO NOT DELETE

su-sws-jira opened the pull request.

su-sws-jira committed:
8.2.16

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr676-1zqvo40sgp02ul1kvwejfptrl4qmx4yt.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f2af44d77d020ff9a654a33

Visual Diffs:
- https://dashboard.tugboat.qa/5f2af44d77d020ff9a654a33/visualdiffs

pookmish committed:
HSD8-871 Added styles dropdown to minimal wysiwyg

pookmish committed:
updated dependencies (#678)

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr676-1zqvo40sgp02ul1kvwejfptrl4qmx4yt.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f2af44d77d020ff9a654a33

Visual Diffs:
- https://dashboard.tugboat.qa/5f2af44d77d020ff9a654a33/visualdiffs

su-sws-jira committed:
Updated dependencies Aug 12 2020

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr676-1zqvo40sgp02ul1kvwejfptrl4qmx4yt.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f2af44d77d020ff9a654a33

Visual Diffs:
- https://dashboard.tugboat.qa/5f2af44d77d020ff9a654a33/visualdiffs

kaseybon committed:
Sparkbox Sprint 22 (#681)

* fix(STN-443): Sprint 20 QA (#665)

* fix(STN-443): email link hover state

* fix(STN-443): remove extra spacing in pill lists

* fix(STN-443): Fix background gradient of vertical linked card with no image in a well

* chore(STN-467): Setup backstop.js for sandbox site

* fix(STN-443): raised cards should have white background to help them stand out in wells

* refactor(STN-443): clean up the spacing system to allow for elements to be nested in rows without adding extra spacing

* refactor(STN-443): cleanup and a little bit of padding on grid items

* chore(STN-443): compiled CSS

Co-authored-by: Kasey Bonifacio <kasey@heysparkbox.com>

* fix(STN-477): subcontent has padding when nothing in it

* feat: add theme setting to toggle site animation enhancements (#675)

JIRA STN-458 https://sparkbox.atlassian.net/browse/STN-458

* fix(STN-476) :  Footer Block Description Configuration (#679)

* fix: add preprocess hook to humsci basic theme to change the block label to block description
* feat: implement review feedback

- add clarifying comments
- change variable name so it is more descriptive

JIRA STN-476 https://sparkbox.atlassian.net/browse/STN-476

* chore: backstop update (#672)

* feat(STN-489): switches backstop to stage env

* docs: add information about syncing from staging

* feat: added new color pairings to the colorful theme (#677)

* feat: added new color pairings to the colorful theme

- Added Lake
- Added Canyon
- Added Cliff

* chore: update colorful, airy theme heading hover style

Updated the colorful link style for the colorful theme, this applies to all color pairings, to account for better accessibility when adding Lake and Canyon's color pairings.

* update: change lake hero overlay and tertiary reversed

* feat(STN-471): Banner (#673)

* feat(STN-471): banner component

* refactor(STN-471): banner and spotlight should use the spacing system refactor introduced in #665

* fix(STN-471): Banner images need to fill the entire width of the available space

* test(STN-471): tries out new behat protocol for banner

* fix(STN-471): circleCI adjustment

* chore: compiled css

Co-authored-by: Lindsay Silver <lindsay@heysparkbox.com>

* fix(STN-507): Fix Spacing Regressions (#682)

* fix(STN-507): carousel and page hero spacing

* fix(STN-507): decrease accordion spacing

* chore(STN-507): fix events link for backstop.js

* fix(STN-507): remove margin top for carousel in hero area

* fix(STN-507): spacing around js pager button

* refactor(STN-507): remove !important where possible, comment where not possible

* doc(STN-507): comment update

* chore(STN-507): compiled CSS

Co-authored-by: Lindsay Silver <lindsay@heysparkbox.com>

* chore: responding to review feedback

Co-authored-by: Lindsay R. Silver <lindsay@heysparkbox.com>
Co-authored-by: Heather Taylor <heather@heysparkbox.com>
Co-authored-by: KelbelR <kelbel@heysparkbox.com>

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr676-1zqvo40sgp02ul1kvwejfptrl4qmx4yt.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f2af44d77d020ff9a654a33

Visual Diffs:
- https://dashboard.tugboat.qa/5f2af44d77d020ff9a654a33/visualdiffs

pookmish committed:
HSD8-873 Disable rules to fix conditions issues

pookmish committed:
HSD8-876 Fix publication ECK display

pookmish committed:
HSD8-881 Remove body trimming on postcard display

pookmish committed:
HSD8-884 Unlock private page components form

pookmish commented:
[Tugboat](https://tugboat.qa) has finished building the preview for this pull request!

Link:
- https://pr676-1zqvo40sgp02ul1kvwejfptrl4qmx4yt.tugboat.qa


Dashboard:
- https://dashboard.tugboat.qa/5f2af44d77d020ff9a654a33

Visual Diffs:
- https://dashboard.tugboat.qa/5f2af44d77d020ff9a654a33/visualdiffs
